package.json

```diff
@@ -12,13 +12,12 @@
         "lerna:run": "lerna run",
         "lerna:rebuild": "lerna run clean; lerna run build;",
         "lerna:publish":
-            "yarn install; lerna run clean; lerna run build; lerna publish --registry=https://registry.npmjs.org/"
+            "yarn install; yarn lerna:rebuild; node ./packages/monorepo-scripts/lib/publish.js"
```

Comment:
is there we can call `yarn publish` in the context of the `monorepo-scripts` package? if not, the two implementations may diverge

Reply:
Can you elaborate?

Reply:
there is a `script:publish` command in the `monorepo-scripts` package, we have to remember to modify both this one and that one